lib/cucumber/runtime.js

```diff
@@ -7,15 +7,20 @@ function Runtime(configuration) {
     start: function start(callback) {
       if (typeof(callback) !== 'function')
         throw new Error(Runtime.START_MISSING_CALLBACK_ERROR);
-      var isStrictRequested;
+      var isStrictRequested, isDryRunRequested;
       try {
         isStrictRequested    = configuration.isStrictRequested();
       } catch(e) {
         isStrictRequested    = false;
       }
+      try {
+        isDryRunRequested    = configuration.isDryRunRequested();
+      } catch(e) {
+        isDryRunRequested    = false;
+      }
```

Comment:
I don't see any reason for the try/catch. Looks like you were just following the style of isStrictRequested, but I believe that can be removed too.